plugins/wasi_nn_rust/src/lib.rs

```diff
@@ -0,0 +1,382 @@
+mod helper;
+mod models;
+
+pub enum ErrNo {
```

Comment:
Is it possible to re-use [error.rs](https://github.com/second-state/wasmedge-wasi-nn/blob/ggml/rust/src/error.rs) from wasmedge-wasi-nn?

Reply:
I tried it out, but I found that there is no explicit method to convert it to i32. Therefore, I still need to manually write the conversion function in the plugin layer (the code on both sides will still decoupled in the future). And it also lacks a Success enum. Given these reasons, it is less concise than declaring a new enum directly, so I prefer to keep the existing approach.

Reply:
Sounds reasonable.